Extension/src/packageVersion.ts

```diff
@@ -66,13 +78,18 @@ export class PackageVersion {
                 diff = this.patch - other.patch;
                 if (diff) {
                     return diff > 0;
-                } else if (this.suffix) {
-                    if (!other.suffix) {
-                        return false;
-                    }
-                    return (this.suffixVersion > other.suffixVersion);
                 } else {
-                    return other.suffix ? true : false;
+                    if (!suffixStr) {
+                        return false; // isEqual = true
```

Comment:
Please update the comment to explain that when `suffixStr` is empty that we only compare the major/minor/patch components of the PackageVersion.